extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyStandaloneBuildStep.java

```diff
@@ -158,6 +160,29 @@ public void boot(ShutdownContextBuildItem shutdown,
         recorder.start(shutdown, requireVirtual.isPresent());
     }
 
+    private static boolean foundOtherCustomExMapper(IndexView index) {
```

Comment:
```suggestion
    private static boolean foundOtherCustomExceptionMapper(IndexView index) {
```

Cosmetic one, might be easier to read later...